### PR TITLE
Changes to reflect new location on the website (Name & Address of location of bootcamp)

### PR DESCRIPTION
**Pull request by Ulaize**
The previous location (Griffin Camp) is no longer where we work. Hence, I have changed the footer name to "La Commune"

### CODE REVIEW

File: src/_layouts/default.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h1 class="main-footer__subscribe-heading">Newsletter</h1>
       <div class="main-footer__twitter"><a href="https://twitter.com/decodemtl" target="_blank"><i class="fa fa-twitter-square fa-2x"></i></a></div>
       <div class="main-footer__email"><a href="mailto:hello@decodemtl.com"><i class="fa fa-envelope-o fa-2x"></i></a></div>
   </div>
-  <div class="main-footer__contact">You can find us at the <a href="http://griffincamp.com/" target="_blank">GriffinCamp</a>,  reach us by
+  <div class="main-footer__contact">You can find us at the <a href="http://griffincamp.com/" target="_blank">La Commune</a>,  reach us by
```
Comment (ziad-saab) [inline src/_layouts/default.html]: Can you please change the link also?

**Comment by ziad-saab** [comment]
Your pull request title could be more descriptive. Remember, in a real project there will often be many pull requests. Having a verbose title is appreciated by those who will maintain the project.

This is a great start. I forgot to mention that there is another "griffin camp" somewhere in the codebase. Can you find it and modify it as well please?

**Comment by Ulaize** [comment]
I have changed the name of the request to be more precise. Better?

I have also changed the location everywhere I could find it as well as the corresponding address.

File: src/about/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of the LaCommune a leading startup accelerator in downtown Montreal's innovation district.
```
Comment (ziad-saab) [inline src/about/index.html]: you changed the name, but you kept the same description. can you please modify the description to reflect what La Commune actually is?

**Comment by Ulaize** [comment]
I changed the description. I hope that worked :)

**Comment by nyanofthemoon** [comment]
Almost there, just missing a space! Please change "LaCommune" to "La Commune".

**Comment by nyanofthemoon** [comment]
Oh, found an extra "the" in "taught at the LaCommune". Please review carefully, so that this pull request is typo-free so that it could make it officially to production (live).

**Comment by Ulaize** [comment]
Hopefully, that is done. Thank you Pusheen :smile:

**Comment by nyanofthemoon** [comment]
"LaCommune" mention are still found in files: "fulltime_faq.json", "faq.json" and "index.html". Maybe just forgot to git add?

**Comment by Ulaize** [comment]
I do not know. I thought I changed that. Probably. OH well, another day. This will not be the branch merged that is for sure :smile:

**Comment by Ulaize** [comment]
Hope that is it?

Thanks for your patience :smiley_cat:

**Comment by nyanofthemoon** [comment]
Almost there! Only missing from files "src/_data/faq.json" and "src/_data/fulltime_faq.json". :+1:

**Comment by Ulaize** [comment]
ugh. I forgot to save.

http://memeshare.net/memes/15/14038.png

**Comment by nyanofthemoon** [comment]
Great job! :+1:

**Pull request closed without merging**